estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,71 @@
+from odoo import api, fields, models
+from odoo.exceptions import UserError
+
+
+class EstatePropertyOffer(models.Model):
+    _name = 'estate.property.offer'
+    _description = 'All offers'
+    _order = 'price desc'
+
+    price = fields.Float(required=True)
+    status = fields.Selection(
+        string='Status',
+        selection=[
+            ('accepted', 'Accepted'),
+            ('refused', 'Refused'),
+        ],
+    )
+    partner_id = fields.Many2one('res.partner', string='Partner')
+    property_id = fields.Many2one('estate.property', string='Property')
+    validity = fields.Integer(default=7)
+    date_deadline = fields.Date(compute='_compute_deadline', inverse="_inverse_deadline")
```

Comment:
```suggestion
    date_deadline = fields.Date(compute='_compute_date_deadline', inverse="_inverse_date_deadline")
```
When comes to naming the compute and inverse method it is always better to use the whole field name